plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/explorer/OpenShiftExplorerView.java

```diff
@@ -123,7 +123,8 @@ private Control createExplanationPane(Control connectionsPane, PageBook pageBook
 	private SelectionAdapter onExplanationClicked(final Control connectionsPane, final Control explanationPane) {
 		return new SelectionAdapter() {
 			public void widgetSelected(SelectionEvent e) {
-				ConnectionWizard wizard = new ConnectionWizard();
+				ConnectionWizard wizard = new ConnectionWizard();				
+				wizard.setWindowTitle("New OpenShift Connection");
```

Comment:
@mlabuda this is the right API-call but I would directly put into the constructor of the ConnectionWizard (https://github.com/jbosstools/jbosstools-openshift/blob/master/plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizard.java#L41) so that this also gets fixed for all the other cases